pkg/cmd/pipeline/logs.go

```diff
@@ -182,9 +191,14 @@ func (opts *logOptions) getAllInputs() error {
 }
 
 func (opts *logOptions) askRunName() error {
+	if opts.limit <= 0 {
```

Comment:
Validation here is fine but cobra already supports https://github.com/spf13/cobra#positional-and-custom-arguments 

Since the validation of options repeat, it may be better to have a `func (opts *logOptions validate() error`

Reply:
I added a `validate()` function in my latest push. I agree with you though that this should eventually be switched to cobra's validation. If it's all right with you, I would like to do this in a separate pr as I would also like to refactor some things around how sorting is handled for `pipelineruns` across the CLI. The switch over to better validation can be part of that, and I can outline it in an issue.